solvers/jaxopt_gd.py

```diff
@@ -52,7 +55,13 @@ def set_objective(self, f_train, f_val, n_inner_samples, n_outer_samples,
         )
 
         @partial(jax.jit, static_argnames=("f", "n_steps"))
-        def inner_solver_fun(outer_var, inner_var, f=None, n_steps=1, lr=.1):
+        def inner_solver_fun(outer_var, inner_var, f=None, n_steps=1):
+            """
+            Solver used to solve the inner problem.
+            The output of this function is differentiable w.r.t. the
+            outer_variable. The Jacobian is computed using implicit
+            differentiation with a conjugate gradient solver.
+            """
```

Comment:
Classical formating for docstring is with a single line short description first, one empty line and then the details:
```suggestion
            """Solver used to solve the inner problem.

            The output of this function is differentiable w.r.t. the
            outer_variable. The Jacobian is computed using implicit
            differentiation with a conjugate gradient solver.
            """
```